@navikt/core/react/src/table/DataCell.tsx

```diff
@@ -17,18 +23,32 @@ export interface DataCellType
   > {}
 
 export const DataCell: DataCellType = forwardRef(
-  ({ className, children = "", align, ...rest }, ref) => {
+  (
+    { className, children = "", align, hasInteractiveContent = true, ...rest },
```

Comment:
Betyr dette at default behaviour er at man ikke kan ekspandere column? (som er motsatt fra default tidligere? hvor mindre man hadde noe annet en td / th?). Burde dette være motsatt?

Reply:
Det mest vanlige er kanskje å ikke ha interactive cells? 🤔

Reply:
Ja tenker at hvis man først har gjort valget med å ha expandableRows så vil man kunne åpne dem by default. Men da må vel `hasInteractoveContent` være `false` by default?

Reply:
Tror det kan være skummelt å anta at bruker sjekker at de har lagt på `hasInteractoveContent={true}` hvis de legger inn f.eks en knapp. Er raskt fare for at de glemmer det av da 🤔 Tilfellene der man vil trenge `hasInteractiveContent` er relavtivt få, og hvis den er default `true` vil de fleste vil forhåpentligvis ikke trenge å forholde seg til den. 

Vil også ikke brekke noen som bruker ExpandableRow i dag slik at vi slipper en major-versjon

Reply:
Men `hasInteractiveContent` er kanskje litt missvisene da.
- `possibleInteractiveContent`
- `canHaveInteractiveContent`
eller noe lignende kan hende er bedre da 🤔  Men ingen av dem fungerte så bra heller...